### DISCUSSION TITLE
Get-AutomationVariable localy

### DISCUSSION THREAD

**Discussion by Seidlm** [microsoft/vscode-azureAutomation#18] [Ideas]
Most of my Scripts are using Variable sin AA, but when I trigger the Script locally, this command fails, causing "Get-AutomationVariable" ist not known.

Any Idea or Update that this can also work locally?

**Comment by fbodmer** [comment]
If you work with PowerShell 5.1 you require the following modules:
0.2.4.2    AzureAutomationAuthoringToolkit
2.0.0.0    PSWorkflow

Also, need to ensure that in C:\Users\<user>\AzureAutomationAuthoringToolkitConfig.json you point to the correct Json Asset files.

PowerShell 7 is a bit tricky. It should work if Orchestrator.AssetManagement.Cmdlets is installed. However, 7.1 has a bug, and 7.2 is not supporting hybrid execution (it's also not available in all regions).

**Reply by Seidlm** [reply]
Is there Documentation or BlogPost for those Steps?

**Comment by Shruti-MS** [comment]
Hi @Seidlm in v1.0.8 we have fixed this. You can access the non-encrypted Variables through "Get-AutomationVariable" command.
Do try and provide us feedback on same.
For encrypted variables we are working on it, will update on this.

Thanks
